source/specify-documents-to-return.txt

```diff
@@ -0,0 +1,179 @@
+.. _java-rs-specify-documents-to-return:
+
+===========================
+Specify Documents to Return
+===========================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :keywords: read, paginate, pagination, order, code example
+
+Overview
+--------
+
+In this guide, you can learn how to specify which documents to return from a read
+operation by using the following methods:
+
+- ``limit()``: Specifies the maximum number of documents to return from a query.
+- ``sort()``: Specifies the sort order for the returned documents.
+- ``skip()``: Specifies the number of documents to skip before returning query results.
+
+Sample Data
+~~~~~~~~~~~
+
+The examples in this guide use the ``sample_restaurants.restaurants`` collection
+from the :atlas:`Atlas sample datasets </sample-data>`. To learn how to create a
+free MongoDB Atlas cluster and load the sample datasets, see the
+:ref:`<java-rs-getting-started>`.
+
+.. include:: includes/reactor-note.rst
+
+Limit
+-----
+
+To specify the maximum number of documents returned from a read operation,
+chain the ``limit()`` method to a ``find()`` method and pass the number
+of documents you want to return.
+
+The following example finds all restaurants that have a ``cuisine`` field value
+of ``"Italian"``, and limits the results to 5 documents:
```

Comment:
[suggestion] Can style this to use the number 5 as you've done, that way it's consistent with style guide the way you've written out "five" in the next paragraph: 
```suggestion
of ``"Italian"``, and limits the results to ``5`` documents:
```